### ISSUE TITLE
Add Blurple emoji reactions to all blurplify commands

### ISSUE THREAD

**Issue by minalike** [python-discord/sir-robin#27]
Once the `blurplify` formatter is merged (#25), I think it would be fun to add the Blurple emoji as a reaction to the output messages for this command.

The emoji ID is 959181314152607794

Reactions are very tiny though, so I'm open to other ideas.

This will be nice to have done by April Fools, but by no means a requirement.
